rtgui/editorpanel.cc

```diff
@@ -46,6 +46,8 @@ class EditorPanel::MonitorProfileSelector
     void prepareProfileBox ()
     {
         profileBox.set_size_request (100, -1);
+        profileBox.setPreferredWidth(90, 150);
```

Comment:
I don't saw this requirement anywhere, so maybe it was only an impression ? Sometimes I align some functions in header files, but it's purely cosmetic. I ran editorpanel.cc through astyle and it hasn't changed that, so this doesn't seem to be important. Some devs add one, but I don't. I'm using the autocompletion which doesn't add space here neither, but I don't think it's important.